obd_ontology/app.py

```diff
@@ -64,6 +64,18 @@ def get_session_variable_list(name: str) -> list:
     return session.get(name)
 
 
+def check_for_invalid_characters(input_string: str) -> bool:
```

Comment:
I'd suggest to call it `invalid_characters` -> improves readability in if-statements etc.